custom_components/ha_washdata/profile_store.py

```
@@ -1424,16 +1763,36 @@ async def add_reference_cycle(
             },
         }
         if meta.get("sampling_interval"):
-            cycle["sampling_interval"] = float(meta["sampling_interval"])
+            try:
+                cycle["sampling_interval"] = float(meta["sampling_interval"])
+            except (TypeError, ValueError):
+                pass  # ignore a malformed sampling_interval rather than raise mid-import
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(TypeError, ValueError)` instead of `try`-`except`-`pass`

(SIM105)